compose/dashboard_conf/nginx.conf

```diff
@@ -0,0 +1,21 @@
+server {
+    listen 80;
+    # gzip config
+    gzip on;
```

Comment:
why do we need to enable `gzip` by default? I think `off` is better for us.

only CDN need to open `gzip` by default.

Reply:
It's ok to use gzip here

https://github.com/ant-design/ant-design-pro/blob/master/docker/nginx.conf#L4

Reply:
use ant-design default configuration temporarily